core/modules/utils/utils.js

```diff
@@ -336,6 +336,10 @@ exports.formatTitleString = function(template,options) {
 	return result;
 };
 
+exports.isValidDate = function(date) {
```

Comment:
I assume that we can't use the existing `$tw.utils.isDate()` method?

Reply:
Thank you for the hint. I haven't found an `isDate` method in `utils.js` and thought there wasn't any available. I'll switch over to that one.